compiler/src/iree/compiler/Dialect/HAL/Target/LLVM/LLVMIRPasses.cpp

```diff
@@ -92,9 +92,6 @@ LogicalResult runLLVMIRPasses(const LLVMTargetOptions &options,
             llvm::AddressSanitizerOptions Opts;
             bool moduleUseAfterScope = false;
             bool useOdrIndicator = false;
-            modulePassManager.addPass(
-                llvm::RequireAnalysisPass<llvm::ASanGlobalsMetadataAnalysis,
-                                          llvm::Module>());
```

Comment:
Please note:
```
commit dacfa24f75c328ae30b710ecadaa18e4ba10cdc6
Author: Mitch Phillips <31459023+hctim@users.noreply.github.com>
Date:   Mon Jun 27 14:40:03 2022 -0700

    Delete 'llvm.asan.globals' for global metadata.
    
    Now that we have the sanitizer metadata that is actually on the global
    variable, and now that we use debuginfo in order to do symbolization of
    globals, we can delete the 'llvm.asan.globals' IR synthesis.
    
    This patch deletes the 'location' part of the __asan_global that's
    embedded in the binary as well, because it's unnecessary. This saves
    about ~1.7% of the optimised non-debug with-asserts clang binary.
    
    Reviewed By: vitalybuka
    
    Differential Revision: https://reviews.llvm.org/D12791
```